runtime/bindings.m

```diff
@@ -88,3 +89,54 @@
 #endif
 #endif
 
+/*
+ * Vector c bindings
+ */
+
+typedef CGPoint (*vision_func) (vector_float2 faceLandmarkPoint, CGRect faceBoundingBox, size_t imageWidth, size_t imageHeight);
+
+CGPoint
+xamarin_CGPoint__VNNormalizedFaceBoundingBoxPointForLandmarkPoint_Vector2_CGRect_nuint_nuint (struct Vector2f faceLandmarkPoint, CGRect faceBoundingBox, xm_nuint_t imageWidth, xm_nuint_t imageHeight) {
+
+	static vision_func func = NULL;
+
+	if (func == NULL) {
+		void *vision_handle = dlopen ("/System/Library/Frameworks/Vision.framework/Vision", RTLD_LAZY);
+		if (vision_handle == NULL)
+			xamarin_assertion_message ("Could not open Vision Framework");
+
+		func = (vision_func) dlsym (vision_handle, "VNNormalizedFaceBoundingBoxPointForLandmarkPoint");
```

Comment:
The header problem can be solved, but the worse problem is that if we don't use dlsym, every time we link with libxamarin.a (which most apps do, at least for release builds) we need to link (weakly) with the Vision framework, which won't work with older Xcodes.